db2 build failure on com.oltpbenchmark.benchmarks.epinions.TestEpinionsLoader

Ran 
```
./mvnw clean package -P db2
```

Error on 

```
[INFO ] 2023-03-27 21:19:23,868 [main]  com.oltpbenchmark.util.ThreadUtil runLoaderThreads - Finished executing 50 Loader Threads [time=0.33s]
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.892 sec <<< FAILURE!
testLoad(com.oltpbenchmark.benchmarks.epinions.TestEpinionsLoader)  Time elapsed: 0.891 sec  <<< FAILURE!
java.lang.AssertionError: No tuples were inserted for table trust
        at com.oltpbenchmark.api.AbstractTestLoader.validateLoad(AbstractTestLoader.java:92)
        at com.oltpbenchmark.api.AbstractTestLoader.testLoad(AbstractTestLoader.java:51)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at junit.framework.TestCase.runTest(TestCase.java:177)
        at junit.framework.TestCase.runBare(TestCase.java:142)
        at junit.framework.TestResult$1.protect(TestResult.java:122)
        at junit.framework.TestResult.runProtected(TestResult.java:142)
        at junit.framework.TestResult.run(TestResult.java:125)
        at junit.framework.TestCase.run(TestCase.java:130)
        at junit.framework.TestSuite.runTest(TestSuite.java:241)
        at junit.framework.TestSuite.run(TestSuite.java:236)
        at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:90)
        at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:252)
        at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:141)
        at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:112)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:568)
        at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
        at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
        at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
        at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
        at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)

Running com.oltpbenchmark.benchmarks.epinions.TestEpinionsBenchmark
```

Failure report at the end

```
Results :

Failed tests:   testLoad(com.oltpbenchmark.benchmarks.epinions.TestEpinionsLoader): No tuples were inserted for table trust

Tests run: 215, Failures: 1, Errors: 0, Skipped: 5

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:41 min
[INFO] Finished at: 2023-03-27T21:20:31-04:00
[INFO] ------------------------------------------------------------------------
```

--- a/src/main/java/com/oltpbenchmark/benchmarks/epinions/EpinionsLoader.java
+++ b/src/main/java/com/oltpbenchmark/benchmarks/epinions/EpinionsLoader.java
@@ -108,10 +108,15 @@ public class EpinionsLoader extends Loader<EpinionsBenchmark> {
             final int lo = i * loadPerThread;
             final int hi = Math.min(this.num_users, (i + 1) * loadPerThread);
 
+            // To ensure that we don't wind up with an empty trust table for
+            // small scales we have the first loader thread ensure that at
+            // least a single trust record is inserted.
+            final boolean firstLoader = (i == 0);
+
             threads.add(new LoaderThread(this.benchmark) {
                 @Override
                 public void load(Connection conn) throws SQLException {
-                    loadTrust(conn, lo, hi);
+                    loadTrust(conn, lo, hi, firstLoader);
                 }
 
                 @Override
@@ -309,7 +314,7 @@ public class EpinionsLoader extends Loader<EpinionsBenchmark> {
      * Zipfian distribution Trusted users are not correlated to heavy
      * reviewers (drawn using a scrambled distribution)
      */
-    public void loadTrust(Connection conn, int lo, int hi) throws SQLException {
+    public void loadTrust(Connection conn, int lo, int hi, boolean firstLoader) throws SQLException {
         Table catalog_tbl = this.benchmark.getCatalog().getTable("trust");
         String sql = SQLUtil.getInsertSQL(catalog_tbl, this.getDatabaseType());
 
@@ -324,6 +329,12 @@ public class EpinionsLoader extends Loader<EpinionsBenchmark> {
             for (int i = lo; i < hi; i++) {
                 long timestamp = System.currentTimeMillis();
                 int trust_count = numTrust.nextInt();
+                // To avoid having an empty trust table during small scale
+                // tests, we make sure that at least one trust record is
+                // inserted to the table.
+                if (firstLoader && i == lo) {
+                    trust_count = Math.max(trust_count, 1);
+                }
                 for (int tc = 0; tc < trust_count; ) {
                     int u_id = reviewed.nextInt();
                     if (!trusted.contains(u_id)) {

